dashboard/README.md

```diff
@@ -1,15 +1,131 @@
+- **Industry professionals**, who can use the insights for analysing trends and understanding the gaming market.
+
+The dashboard is built using Streamlit, allowing for interactive and responsive visualisations that help users explore trends across platforms.
+
+## Features
+
+- Visualise game releases across platforms with customizable filters.
+- Subscribe to weekly reports and genre-specific game updates via email.
+- Explore genre popularity, operating system trends, and more.
+
 ## Wireframe
 
-1. Dashboard (graphs that will be customised depending on user filters)
+1. **Dashboard** (graphs that will be customised depending on user filters)
 
 ![Dashboard](wireframe_dashboard.png)
 
-2. About (information about the project and dashboard)
+2. **About** (information about the project and dashboard)
 
 ![About](wireframe_about.png)
 
-3. Subscribe (information about subscribing to the mailing list)
+3. **Games** (list of games with search functionality)
+
+![Games](wireframe_games.png)
+
+4. **Subscribe** (options to subscribe to relevant emails)
 
 ![Subscribe](wireframe_subscribe.png)
+
+## Installation and Setup
+
+### Repository Structure
+
+The application files are structured as follows:
+
+- `Dashboard.py`: The main Streamlit dashboard file that displays the graphs and insights.
+- `sl_queries.py`: Contains SQL query functions to retrieve data from the RDS instance.
+- `functions.py`: Functions for creating charts and preprocessing data.
+- `test_dashboard.py`: For testing the dashboard functionalities.
+- `pages/`: Contains additional pages for the dashboard.
+    - `About.py`: Information about the Games Tracker project.
+    - `Games.py`: Displays a searchable table of games.
+    - `Subscribe.py`: Subscription page to manage user notifications.
+    - `pages/utils/subscribe_functions.py`: Utility functions for subscribing/unsubscribing users.
+
+### Environment Variables
+
+Ensure the following environment variables are defined in your .env file:
+```
+DB_HOST=<Your RDS host>
+DB_PORT=<Your RDS port>
+DB_USER=<Your database user>
+DB_PASSWORD=<Your database password>
+DB_NAME=<Your database name>
+REGION=<Your AWS region>
+AWS_ACCESS_KEY=<Your AWS access key>
+AWS_SECRET_KEY=<Your AWS secret key>
+```
+
+### Installing Dependencies
+
+To set up the project:
+
+1. Create a virtual environment:
+ ```   
+python -m venv .venv
+source .venv/bin/activate  # For Linux/MacOS
+.venv\Scripts\activate     # For Windows
+``` 
+
+2. Install the required dependencies:
+```
+pip install -r requirements.txt
+```
+
+### Running the Application
+
+Run the Streamlit dashboard with:
+```
+streamlit run Dashboard.py
+```
+
+### Docker Setup and Deployment
+
+Build, tag, push, and deploy your Dockerised dashboard on AWS:
+
+1. Dockerfile
+
+Ensure the repository contains a Dockerfile that defines how to build the application image.
+2. Build the Docker image with platform
+```
```

Comment:
We could write a bash file like in the other directories: `push-to-ecr` which handles these steps as one command - just for consistency.